<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowConductivity</class>
 <widget class="QMainWindow" name="MainWindowConductivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>141</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Фракталы</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,10">
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_Logic">
      <property name="rightMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_draw">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #E0E0E0, stop: 1 #FFFFFF);
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }</string>
        </property>
        <property name="title">
         <string>Отрисовка</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_draw">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonDragon">
           <property name="text">
            <string>Дракон Хайтера-Хайтвея</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonMatSerpinsky">
           <property name="text">
            <string>Ковер Серпинского</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSerpinsky">
           <property name="text">
            <string>Кривая Серпинского</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonHylbert">
           <property name="text">
            <string>Кривая Гильберта</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonHosper">
           <property name="text">
            <string>Кривая Госпера</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonPeano">
           <property name="text">
            <string>Кривая Пеано</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonBush">
           <property name="text">
            <string>Куст</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonMosaic">
           <property name="text">
            <string>Мозаика</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonIsland">
           <property name="text">
            <string>Остров</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSnowflake">
           <property name="text">
            <string>Снежинка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSnowflakeKosh">
           <property name="text">
            <string>Снежинка Коша</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonWeed">
           <property name="text">
            <string>Сорняк</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonFlower">
           <property name="text">
            <string>Цветок</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonChain">
           <property name="text">
            <string>Цепочка</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxPoints">
           <property name="text">
            <string>Рисовать точками</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_split">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="leftMargin">
            <number>2</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_X">
             <property name="text">
              <string>Уровень</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxLevel</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBoxLevel">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_surface">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBoxLevel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
